<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ScreenSettings</class>
 <widget class="QWidget" name="W_ScreenSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar
{
	border: none;
	background-color:rgba(250,250,250,50);
	width:10px;
	margin: 0px 0px 0px 0px;
	border-radius:5px;
}

QWidget
{
background-color:rgba(0,0,0,0);
}

QFrame
{
	border: none;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>578</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>28</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	color:white;
}</string>
             </property>
             <property name="text">
              <string>PARAMETREs DE CONNEXION</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="input_connectionState">
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">QFrame
{
	color:white;
	background-color:white;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	color:white;
}</string>
                 </property>
                 <property name="text">
                  <string>Nom</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	color:white;
}</string>
                 </property>
                 <property name="text">
                  <string>Adresse IP</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	color:white;
}</string>
                 </property>
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLineEdit" name="input_name">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	background-color:rgba(250,250,250,3);
	border-bottom: 1px solid rgba(250,250,250,150);
	border-top: 0px solid white;
	border-left: 0px solid white;
	border-right: 0px solid white;
	color:white;
	padding:0px;
}

QLineEdit::hover{
	background-color:rgba(250,250,250,8);
	border-bottom: 1px solid rgba(250,250,250,200);
	color:white;
}</string>
                 </property>
                 <property name="text">
                  <string>MyName</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="input_ip">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	background-color:rgba(250,250,250,3);
	border-bottom: 1px solid rgba(250,250,250,150);
	border-top: 0px solid white;
	border-left: 0px solid white;
	border-right: 0px solid white;
	color:white;
}

QLineEdit::hover{
	background-color:rgba(250,250,250,8);
	border-bottom: 1px solid rgba(250,250,250,200);
	color:white;
}</string>
                 </property>
                 <property name="text">
                  <string>127.0.0.1</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="input_port">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Hogwarts</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QSpinBox{
	background-color:rgba(250,250,250,3);
	border-bottom: 1px solid rgba(250,250,250,150);
	border-top: 0px solid white;
	border-left: 0px solid white;
	border-right: 0px solid white;
	color:white;
}

QSpinBox::hover{
	background-color:rgba(250,250,250,8);
	border-bottom: 1px solid rgba(250,250,250,200);
	color:white;
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="maximum">
                  <number>65535</number>
                 </property>
                 <property name="value">
                  <number>35000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="buttonConnect">
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color:rgba(250,250,250,20);
	color:white;
	border-radius:3px;
	padding-bottom:4px;
}
QPushButton:hover{
	background-color:rgba(250,250,250,40);
}

QPushButton:pressed{
	background-color:rgba(250,250,250,100);
}</string>
             </property>
             <property name="text">
              <string>Connexion</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>137</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Hogwarts</family>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color:white;
}</string>
           </property>
           <property name="text">
            <string>PARAMETRES SONORES</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="styleSheet">
            <string notr="true">QFrame
{
	color:white;
	background-color:white;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	color:white;
}</string>
             </property>
             <property name="text">
              <string>Volume principale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="styleSheet">
              <string notr="true">QSlider::groove{
                    border: no border;
                    background-color:rgba(250,250,250,0);
                    height:7px;
                    margin: 0px 0px 0px 0px;
                    border-radius:5px;
}

QSlider::handle{
				background:transparent;
                min-height: 20px;
				width: 20px;
				margin-right:-10px;
}

QSlider::add-page:horizontal {
				background: rgba(250,250,250,20);
				min-width:0px;
}

QSlider::sub-page:horizontal {
				background: rgba(250,200,45,200);
}</string>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="input_sounds">
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox{
	color:white;
}

QCheckBox::indicator:checked {
	background-color : rgba(250,200,45,200);
}
QCheckBox::indicator:checked:hover {
	background-color : rgba(250,200,45,250);
}

QCheckBox::indicator:unchecked {
	background-color : rgba(250,250,250,20);
}
QCheckBox::indicator:unchecked:hover {
	background-color : rgba(250,250,250,40);
}</string>
             </property>
             <property name="text">
              <string>Effets sonores</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="input_music">
             <property name="font">
              <font>
               <family>Hogwarts</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox{
	color:white;
}

QCheckBox::indicator:checked {
	background-color : rgba(250,200,45,200);
}
QCheckBox::indicator:checked:hover {
	background-color : rgba(250,200,45,250);
}

QCheckBox::indicator:unchecked {
	background-color : rgba(250,250,250,20);
}
QCheckBox::indicator:unchecked:hover {
	background-color : rgba(250,250,250,40);
}</string>
             </property>
             <property name="text">
              <string>Musique</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>137</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
